### DISCUSSION TITLE
Why bootstrap back to version 4.5.3?

### DISCUSSION THREAD

**Discussion by pschrey2** [google/docsy#631] [General]
Hi, bootstrap seems to be switched back to version 4.5.3 on May 29 with commit c5f1613fb1b163ad4ea18e603fc1915494f4bf49 (see git submodule bootstrap at https://github.com/google/docsy/tree/master/assets/vendor).
Any reason for that?
Regards, Patrik

**Comment by narrenfrei** [comment]
As the commit https://github.com/google/docsy/commit/c5f1613fb1b163ad4ea18e603fc1915494f4bf49 is from me, I only can say that there was no reason and it only was accident, so far not noticed.

The commit was part of my PR #554 for taxonomy support and it was quite confusing challenge to get the PR merged because of some issues with git and the cla bot. Maybe it happened somehow in all the mess. Sorry, if it was my fault.

**Reply by pschrey2** [reply]
Ok, no problem, those things happen.
Can the submodule itself be refered back to version 4.6.0?

Because now this submodule and this file https://github.com/google/docsy/blob/master/layouts/partials/scripts.html are currently not in sync.
The upgrade was done earlier by @deining on April 12 (https://github.com/google/docsy/commit/8b2f69581cb1a31be1e00c930533f46a37d2abcb#diff-05e01d0fe33a568f907cb8c03ee25e8cef7221d73f052ac04392232753d25467) plus an SRI hash fix later on April 17.

Not sure about your process, does this require an issue first?

**Reply by pschrey2** [reply]
Issue #626 could be related to this

**Reply by narrenfrei** [reply]
> Can the submodule itself be refered back to version 4.6.0?

In my point of view, it shouldn't be a problem or rather it should changed back of course.

@LisaFC  or @deining 
Can you help there?

**Reply by LisaFC** [reply]
Sure, I can change it back.

**Reply by LisaFC** [reply]
Though actually, maybe add an issue just to make sure it's tracked and that we close it (and in case there's any oddities with the other scripts page).

**Reply by pschrey2** [reply]
Done, see #632
